### DISCUSSION TITLE
Can't find filled tabler icons

### DISCUSSION THREAD

**Discussion by LexioDev** [ng-icons/ng-icons#129] [Q&A] [answered]
Hey! First of all, thanks for that job. I coudn't find a way to import and work easily with icons in angular and that is amazing!

I'm facing a problem. I have beeing looking for tabler filled icons but i can't find them. Is there any way to include or use them that i don't know? Or if they aren't included it would be an amazin improvment.

Again, thanks for you job and your help!

**Comment by ashley-hunter** [comment]
> Hey! First of all, thanks for that job. I coudn't find a way to import and work easily with icons in angular and that is amazing!

Thank you! I'm glad it has been useful for you!

> I'm facing a problem. I have beeing looking for tabler filled icons but i can't find them. Is there any way to include or use them that i don't know? Or if they aren't included it would be an amazin improvment.

Sure, I wasn't aware there was a filled set for tabler icons, thanks for letting me know. I will get a release out later today with those icons included.

**Comment by ashley-hunter** [comment, marked as answer]
This should now be available from `@ng-icons/tabler-icons/fill` in v29.2.0 which was just released.
